src/Microsoft.ML.Transforms/OptionalColumnTransform.cs

```diff
@@ -26,7 +26,15 @@
 
 namespace Microsoft.ML.Runtime.DataPipe
 {
-    public class OptionalColumnTransform : RowToRowMapperTransformBase
+    /// <summary>
+    /// This transform is used to mark some of the columns (e.g. Label) optional during training so that the columns are not required during scoring.
+    /// When applied to new data, if optional columns are not present a meta column is created having the same properties (e.g. 'name', 'type' etc.) as used during training.
```

Comment:
>meta column [](start = 72, length = 11)

The phrase meta-column is a bit odd, I'm not sure that's a good description. Maybe dummy column?